argocd/federated-etl-argo/argocd-kubecost-primary.yaml

```diff
@@ -0,0 +1,45 @@
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: kubecost
+  namespace: argocd
+spec:
+  project: default
+  source:
+    chart: cost-analyzer
+    repoURL: https://kubecost.github.io/cost-analyzer/
+    targetRevision: 1.105.7
```

Comment:
1.105.7?

Reply:
oh man, this should definitely be 1.105.1.  Thank you!